source/appModules/powerpnt.py

```diff
@@ -1128,6 +1128,14 @@ def _getFormatFieldAndOffsets(self, offset, formatConfig, calculateOffsets=True)
 			formatField["link"] = True
 		return formatField, (startOffset, endOffset)
 
+	def _setCaretOffset(self, offset: int):
```

Comment:
@coderabbitai could you suggest an implementation for `_setSelectionOffsets`?

Reply:
@coderabbitai please explain why you have done
```
	startChar = textRange.characters(start + 1)
	endChar = textRange.characters(end + 1)
	textRange = textRange.characters(startChar.start, endChar.start - startChar.start)
```
rather than
```
	textRange = textRange.characters(start + 1, end - start)
```

Reply:
@LeonarddeR it is done :)